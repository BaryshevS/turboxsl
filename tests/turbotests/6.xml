<test host="www.litres.ru" chars="5:00:42" chars1="50042" hour="5" min="00" sec="42">
</test>