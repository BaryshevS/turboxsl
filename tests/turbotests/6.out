<?xml version="1.0"?>
<head xmlns:fo="http://www.w3.org/1999/XSL/Format"><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/></head><body xmlns:fo="http://www.w3.org/1999/XSL/Format">
		format-number: 00<br/>
		format-number: 42<br/>
		translate: 50042<br/>
		substring: 5<br/>
		substring: 00<br/></body>
